src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -161,6 +161,17 @@ template <size_t Dim, typename Frame, typename DataType>
 struct WeylElectric : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
+
+/*!
+ * \brief Computes the scalar \f$E_{ij} E^{ij}\f$ from the electric part of the
+ * Weyl tensor \f$E_{ij}\f$ and the inverse spatial metric \f$\gamma^{ij}\f$,
+ * i.e. \f$E_{ij} E^{ij} = \gamma^{ik}\gamma^{jl}E_{ij}E_{kl}\f$.
+ */
+template <typename DataType>
+struct WeylElectricScalar : db::SimpleTag {
+  using type = Scalar<DataType>;
+  static std::string name() noexcept { return "WeylElectricScalar"; }
```

Comment:
you can remove this as the name is auto generated

Reply:
Great, thank you both for the feedback!